lib/Dancer2/CLI/Gen.pm

```diff
@@ -220,29 +218,26 @@ sub _copy_templates {
         }
 
         if( $from !~ m/\.(ico|jpg|png|css|eot|map|swp|ttf|svg|woff|woff2|js)$/ ) {
-            $content = _process_template($content, $vars);
+            $content = $self->_process_template($content, $vars);
         }
 
-        open( my $fh, '>:raw', $to ) or die "unable to open file `$to' for writing: $!";
-        print $fh $content;
-        close $fh;
-
+        path( $to )->spew_raw( $content );
         if( $ex ) {
-            chmod( 0755, $to ) or warn "unable to change permissions for $to: $!";
+            $to->chmod( 0755 ) or warn "unable to change permissions for $to: $!";
         }
     }
 }
 
 sub _create_manifest {
     my ( $self, $files, $dir ) = @_;
 
-    my $manifest_name = catfile( $dir, 'MANIFEST' );
+    my $manifest_name = path( $dir, 'MANIFEST' );
     open( my $manifest, '>', $manifest_name ) or die $!;
     print $manifest "MANIFEST\n";
 
     foreach my $file( @{ $files } ) {
-        my $filename       = substr $file->[1], length( $dir ) + 1;
-        my $basename       = basename $filename;
+        my $filename       = $file->[1]->relative( $dir );
+        my $basename       = path( $filename )->basename;
```

Comment:
this assumes `$file->[1]` is a `Path::Tiny` object, so better:
```
        my $filename       = path( $file->[1] )->relative( $dir );
        my $basename       = $filename->basename;
```

Reply:
Fixed. Thanks!